towncrier_template.j2

```diff
@@ -0,0 +1,26 @@
+{% if render_title %}
+## v{{ versiondata.version }} ({{ versiondata.date }})
+
+{% endif %}
+{% for section, _ in sections.items() %}
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section] %}
+{% if sections[section][category]|length != 0 %}
+### {{ definitions[category]['name'] }}
```

Comment:
Is this autocapitalized? `### Added` or `### added`?

Reply:
It is:

```
nautobot on  gas-2362-release-notes [$]
➜ poetry run towncrier build --version 1.4.3
Loading template...
Finding news fragments...
Rendering news fragments...
Writing to newsfile...
Staging newsfile...
Removing news fragments...
I want to remove the following files:
/Users/bryanculver/Developer/repos/ntc/nautobot/changes/2362.added
Is it okay if I remove those files? [Y/n]: Y
Done!

nautobot on  gas-2362-release-notes [$✘+]
➜ git diff HEAD
changes/2362.added --- Text
1 add change log fragments and automated release notes for nautobot github project

nautobot/docs/release-notes/version-1.4.md --- Text
162 162 The `settings_and_registry` default context processor was changed to purely `settings` - the (large) Nautobot application registry dictionary is no longer provided as part of the render context for all templates by default. Added a new `registry` template tag that can be invoked by specific templates to provide this variable where needed.
163 163
164 164 <!-- towncrier release notes start -->
... 165 ## v1.4.3 (2022-09-08)
... 166
... 167 ### Added
... 168
... 169 - [#2362](https://github.com/nautobot/nautobot/issues/2362) - add change log fragments and automated release notes for nautobot github project
... 170
... 171
165 172 ## v1.4.2 (2022-09-05)
```